app/src/main/java/com/hedvig/app/ApplicationModule.kt

```diff
@@ -618,6 +622,10 @@ val trackerModule = module {
     }
     single { MixpanelTracker(get()) } bind TrackerSink::class
     single { FirebaseTracker(get()) } bind TrackerSink::class
+    single<HAnalytics> { HAnalyticsFacade(getAll<HAnalyticsSink>().distinct()) }
```

Comment:
I used the sources/sinks-based model for `hAnalytics` as well, in order to be able to build for example engineering tooling on top of it.